core/arch/arm/kernel/kern.ld.S

```diff
@@ -53,9 +53,27 @@
 OUTPUT_FORMAT(CFG_KERN_LINKER_FORMAT)
 OUTPUT_ARCH(CFG_KERN_LINKER_ARCH)
 
+/*
+ * Start coherent RAM with a platform specific structure.
+ * Generic data to be place in coherent RAM can use attribute "coherent_ram".
+ */
+#define DECLARE_COHERENT_RAM_SECTION						\
```

Comment:
Aha, I didn't notice that. This is good then.
The `\` at the end of each line seems to have ended up at column 81 which obviously is beyond 80.